docs/htdocs/info/docs/tools/vep/vep_formats.html

```diff
@@ -652,9 +661,7 @@ <h2 id="json"> JSON output</h2>
     VCF input) is reported under the "input" key in order to aid aligning input
     with output. When using a cache file, frequencies for co-located variants are reported by default (see
     <a href="./script/vep_options.html#opt_af_1kg">--af_1kg</a>,
-    <a href="./script/vep_options.html#opt_af_esp">--af_esp</a>,
-    <a href="./script/vep_options.html#opt_af_gnomad">--af_gnomad</a>,
-    <a href="./script/vep_options.html#opt_af_exac">--af_exac</a>).
+    <a href="./script/vep_options.html#opt_af_gnomade">--af_gnomad</a>).
```

Comment:
Shouldn't this be `--af_gnomade`?
```suggestion
    <a href="./script/vep_options.html#opt_af_gnomade">--af_gnomade</a>).
```